src/services/message.service.ts

```diff
@@ -40,6 +43,52 @@ export class MessageService {
     }
   }
 
+  async reactionMessage(
+    message: IMessage,
+    messageInfo: MessageEmbed | string,
+    reactions: IEmojiTable[],
+    lambda: Function
+  ): Promise<IMessage> {
+    const msg: IMessage = await message.reply(messageInfo);
```

Comment:
I would prefer you just pass in a channel, and send it there.
There is not always a message to reply to

Reply:
that's true!

Reply:
Just to note, we will look into this at a later date.